test/ut/sdk/test_compressor_torch.py

```diff
@@ -418,6 +418,39 @@ def test_torch_quantizer_export(self):
             calibration_config = quantizer.export_model(model_path, calibration_path, onnx_path, input_shape, device)
             assert calibration_config is not None
 
+    def test_quantizer_load_calibration_config(self):
+        configure_list = [{
+            'quant_types': ['weight', 'input'],
+            'quant_bits': {'weight': 8, 'input': 8},
+            'op_names': ['conv1', 'conv2']
+        }, {
+            'quant_types': ['output', 'weight', 'input'],
+            'quant_bits': {'output': 8, 'weight': 8, 'input': 8},
+            'op_names': ['fc1'],
+        }, {
+            'quant_types': ['output', 'weight', 'input'],
+            'quant_bits': {'output': 8, 'weight': 8, 'input': 8},
+            'op_names': ['fc2'],
+        }]
+        quantize_algorithm_set = [torch_quantizer.ObserverQuantizer, torch_quantizer.QAT_Quantizer, torch_quantizer.LsqQuantizer]
+        calibration_config = None
+        for quantize_algorithm in quantize_algorithm_set:
+            model = TorchModel().eval()
+            model.relu = torch.nn.ReLU()
+            optimizer = torch.optim.SGD(model.parameters(), lr=0.01, momentum=0.5)
+            quantizer = quantize_algorithm(model, configure_list, optimizer)
+            quantizer.compress()
+            if calibration_config is not None:
```

Comment:
I am not sure it is better to load config after instantiating the quantizer or during instantiate the quantizer

Reply:
Your concern is valid. Currently, I think we can keep this design because we need an `load_calibration_config` function anyhow and we can discuss this problem while consider about refactoring.